homework/homework2/src/App.js

```diff
@@ -54,14 +55,16 @@ function App () {
         deadline: '',
         done: false,
       });
-    } else {window.alert('Please enter a todo');
+    } else {window.alert('Please enter a todo description and select a deadline');
     }
   };
   const [showTodoList, setShowTodoList] = useState(false);
-  const [buttonText, setButtonText] = useState(false);
 
   const toggleList = () => {
-    return setShowTodoList(!showTodoList), setButtonText(!buttonText);
+    return setShowTodoList(!showTodoList);
+  };
+  const onFocusInput = e => {
+    e.currentTarget.type = 'date';
   };
```

Comment:
Why don't you just implement the html date input field? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date